packages/block-library/src/embed/core-embeds.js

```diff
@@ -151,15 +151,17 @@ export const others = [
 			title: 'Crowdsignal',
 			icon: embedContentIcon,
 			keywords: [ 'polldaddy' ],
-			transform: [ {
-				type: 'block',
-				blocks: [ 'core-embed/polldaddy' ],
-				transform: ( content ) => {
-					return createBlock( 'core-embed/crowdsignal', {
-						content,
-					} );
-				},
-			} ],
+			transforms: {
+				from: [ {
+					type: 'block',
+					blocks: [ 'core-embed/polldaddy' ],
+					transform: ( content ) => {
+						return createBlock( 'core-embed/crowdsignal', {
+							content,
+						} );
+					},
+				} ],
+			},
```

Comment:
Not sure if this is a breaking change or not. Simply changed the schema of JSON to make it consistent with other transforms.